### PR TITLE
Added <assignedTo> field in the task creation form for Issue #18

### PR DESCRIPTION
**Pull request by dimky01**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (swarajpure) [inline .gitignore:1]: What's this doing?

Comment (dimky01) [reply .gitignore:1]: Those were files added by intellij that I removed

Comment (swarajpure) [reply .gitignore:1]: Can we remove this `.gitignore` then? It's not needed for the project, right?

Comment (dimky01) [reply .gitignore:1]: Yes, it's not needed, you can remove it

Comment (swarajpure) [reply .gitignore:1]: Can you remove the file from your machine, commit and push again? The commit will get appended this PR itself and the file will not be a part of this PR anymore 🙂

Comment (dimky01) [reply .gitignore]: I've done that
Do I need a fresh PR?

Comment (swarajpure) [reply .gitignore]: Nope. You can see the changes has been reflected in this PR 🙂

**Review by swarajpure — APPROVED** [review]
Thank you for this one!
Congratulations on your first pull request in Real Dev Squad! 🥳

**Pull request merged**